config/config.php

```diff
@@ -26,7 +26,17 @@
 
 $ver_str = 'Nextcloud Client ' . $ver;
 
-$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
+if (version_compare($version, '3.0.3') < 0) {
+	$url = 'https://download.nextcloud.com/desktop/releases/';
+        $linux_url = $url . 'Linux/';
+        $windows_url = $url . 'Windows/';
+        $mac_url = $url . 'Mac/Installer/';
+} else {
+	$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
+        $linux_url = $url;
+        $windows_url = $url;
+        $mac_url = $url;
+}
```

Comment:
Ahh spacing ^^

Reply:
Yeah my vim setup isn't a huge friend of indenting with tabs. Should be fixed on next push.